tests/agents/test_details_agent.py

```
@@ -178,42 +131,59 @@ def test_create_strict_component_subgraph(self):
+        mock_program_graph = MagicMock()
+        mock_program_graph.filter_by_nodes.return_value = mock_subgraph
+        mock_program_graph.induced_by_symbols.return_value = mock_scoped_program_graph
 
         self.mock_static_analysis.get_languages.return_value = ["python"]
-        self.mock_static_analysis.get_cfg.return_value = mock_cfg
+        self.mock_static_analysis.get_program_graph.return_value = mock_program_graph
 
-        subgraph_cluster_results, subgraph_cfgs = agent._create_strict_component_subgraph(self.test_component)
+        with patch("agents.cluster_methods_mixin.HierarchicalInfomapClusterer") as clusterer:
+            clusterer.return_value.cluster.return_value = mock_sub_cluster_result
+            subgraph_str, subgraph_cluster_results, subgraph_cfgs = agent._create_strict_component_subgraph(
```

Ruff (0.16.0)

[warning] Unpacked variable `subgraph_cluster_results` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)